<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudentExam</class>
 <widget class="QWidget" name="StudentExam">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>آزمون</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="questionLabel">
     <property name="text">
      <string>سوال آزمون:</string>
     </property>
   </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="questionTextEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="answerLabel">
     <property name="text">
      <string>پاسخ شما:</string>
     </property>
   </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="answerTextEdit"/>
   </item>
   <item>
    <widget class="QPushButton" name="submitButton">
     <property name="text">
      <string>ارسال</string>
     </property>
   </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
